src/ClassDiscovery.php

```diff
@@ -109,7 +110,7 @@ private static function storeInCache($type, $class)
     /**
      * Set new strategies and clear the cache.
      *
-     * @param DiscoveryStrategy[] $strategies
+     * @param array $strategies string array of FQN to a DiscoveryStrategy
```

Comment:
i'd prefer to write it out anyways :-)

`@param string[] $strategies Fully qualified class name to a DiscoveryStrategy